core/src/test/java/nav/statusplattform/core/repositories/HelpTextRepositoryTest.java

```diff
@@ -182,4 +171,9 @@ private HelpTextEntity getHelpTextEnity(ServiceType serviceType, int number) {
                 .setType(serviceType)
                 .setContent(SampleData.getRandomFromArray(helpTextDescriptions));
     }
+
+    private int randomHelpTextNumber() {
+        Random random = new Random();
+        return random.nextInt(5) + 1; //up to five service and component help texts
+    }
```

Comment:
Instead of hardcoding 1-5, I suggest you let it take an unsigned integer as input.
And if it's null, then you default to 5.

Gives you alot more flexibility.

Reply:
Also, this is not unique to HelpTextRepositoryTest in any way. If you'd like it to be somehow special for HelpTextRepositoryTest (instead of adding `random.nextInt(x)+1` everywhere, which I think could be an acceptable solution too), I'd like to see you;

1. Move `randomHelpTextNumber()` -> to SampleData (named something that has nothing to do w/HelpText, like `randomPositiveInteger(max)`
1. Make `HelpTextRepositoryText` have a saved class member stored as the "max value" you've currently hard-coded to `5` on [line 177](https://github.com/navikt/statusplattform/pull/54/files#diff-78b8e8b76dbdea8fdb0d70290c699e3a216133341078c2fbac381313578ba4b8R177).
1. For each of the other test packages/modules, go through them and replace any code that generates a random number with the usage of the new function referred to in the first point (as a part of this PR).

Reply:
1-3.  ArrayList numbers was deleted from HelpTextRepositoryTest, also randomHelpTextNumber() was removed and the method placed in SampleData. In addition, the method now has a general name, and takes an argument representing the maximum number of entries allowed.